index.html

```diff
@@ -108,23 +132,21 @@ <h4>Course</h4>
 					</tr>
 				</table>
 				<p>An object also has a level which tells a content consumer what academic level your curriculum is intended for. For instance, if you are uploading a object that is intended for K-12, then you should specify the academic level here.  There are not limits to the amount of levels a object can have, you could specify the object to be suitible to anyone (K-12 through Postgraduate).</p>
+				<p>The description should accurately describe what you&#39;re learning object is and briefly state what it hopes to accomplish. This is indexed for searches and shown on search results. Keep in mind that there is a section where you will write out the outcomes for your learning object in greater detail. This section is also equired in order to have your object reviewed and published.</p>
```

Comment:
In the same line it should say "This section is required ..." instead of "This section is also equired"